### PR TITLE
docs: Provide CN translation for commands.rst

### PR DESCRIPTION
**Pull request by Shen765**
## Description

This PR:

- Adjusted some format issues and unclear expressions in commands.rst based on Espressif Style Guide
- Provided CN translation for docs_espressif/en/commands.rst
- TODO: Closes [DOC-9649](https://jira.espressif.com:8443/browse/DOC-9649) once merged

### CODE REVIEW

**Comment by Shen765** [comment]
Hi @zztiger123 @0cici,
Pls help review the changed files. PS You can preview the built HTML files by clicking this [Espressif documentation preview](https://preview-docs.espressif.com/projects/vscode-esp-idf-extension/en/1358-merge/index.html) link automatically generated by bot. Thx!

image

**Review by 0cici — APPROVED** [review]
Hi @Shen765 , the translation looks good! Left some comments, please check~

File: docs_espressif/en/commands.rst [hunk 1]
```diff
@@ -1,53 +1,55 @@
...
-   * - Build, Flash and start a monitor on your device
-     - Build the project, write binaries program to device and start a monitor terminal with a single command. Similar to ``idf.py build flash monitor``
-   * - Build your project
+   * - Add VS Code Configuration Folder
+     - Add **.vscode** files to the currently opened project directory. This includes launch.json (for debugging), settings.json and c_cpp_properties.json for syntax highlight.
+   * - Build, Flash and Start a Monitor on Your Device
+     - Build the project, write binaries program to device and start a monitor terminal with a single command. Similar to ``idf.py build flash monitor``.
+   * - Build Your Project
      - Build your project using ``CMake`` and ``Ninja-build`` as explained in `ESP-IDF Build System Using Cmake Directly <https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/build-system.html#using-cmake-directly>`_. You could modify the behavior of the build task with **idf.cmakeCompilerArgs** for Cmake configure step and **idf.ninjaArgs** for Ninja step. For example, using **[-j N]** where N is the number of jobs run in parallel.
```
[the remark this answers is not in the document]
Comment (Shen765) [reply docs_espressif/en/commands.rst:24]: "using" should be reserved.

File: docs_espressif/en/commands.rst [hunk 2]
```diff
@@ -57,78 +59,78 @@ All commands start with ``ESP-IDF:``.
...
+     - Select the OpenOCD configuration files that match your Espressif device target, such as DevKitC or ESP-Wrover-Kit. This is necessary for flashing with JTAG or debugging your device.
+   * - Select Where to Save Configuration Settings
+     - In Visual Studio Code, settings can be saved in three places: User Settings (global settings), workspace ( .code-workspace file) or workspace folder (.vscode/settings.json).
+   * - Select Output and Notification Mode
+     - This extension shows many notifications and output in the Output window **ESP-IDF**. This command allows you to set if to show notifications only, output only, both notifications and output, or neither.
+   * - Set Espressif Device Target
+     - This will set the target for the current project (IDF_TARGET). Similar to **idf.py set-target**. For example, if you want to use ESP32 or ESP32-C3, you need to execute this command.
    * - Set ESP-MATTER Device Path (ESP_MATTER_DEVICE_PATH)
      - The **ESP-IDF: Set ESP-MATTER Device Path (ESP_MATTER_DEVICE_PATH)** is used to define the device path for ESP-Matter. ESP-Matter is not supported in Windows.
```
[the remark this answers is not in the document]
Comment (Shen765) [reply docs_espressif/en/commands.rst:126]: I think "in Windows" and "on Windows" both are OK. So it is unnecessary to replace this prep. :)

File: docs_espressif/zh_CN/commands.rst [hunk 3]
```diff
@@ -1 +1,136 @@
...
+   * - 添加 VS Code 配置文件夹
+     - 将 **.vscode** 文件添加到当前打开的项目目录中。这些文件包括 launch.json（用于调试）、settings.json 和 c_cpp_properties.json（用于语法高亮）。
+   * - 构建、烧录项目并监视设备
+     - 此命令可用于构建项目、将二进制程序写入设备，并启动监视终端，效果类似于 ``idf.py build flash monitor``。
+   * - 构建项目
+     - 使用 ``CMake`` 和 ``Ninja-build`` 来构建项目，具体说明请参见 `ESP-IDF 构建系统直接使用 CMake <https://docs.espressif.com/projects/esp-idf/zh_CN/latest/esp32/api-guides/build-system.html#cmake>`_。若想修改构建任务的行为，可以在配置 Cmake 时使用 **idf.cmakeCompilerArgs** 命令，或在配置 Ninja 时使用 **idf.ninjaArgs** 命令。例如，可以使用 **[-j N]** 来设置并行作业数，其中 N 是并行作业的数量。
+   * - 清除 eFuse 摘要
+     - 从 ESP-IDF 资源管理器的 EFUSEEXPLORER 中清除 eFuse 摘要集。
+   * - 清除 ESP-IDF 搜索结果
+     - 从ESP-IDF 资源管理器 ``文档搜索结果`` 选项卡中清除所有搜索结果。
```
Comment (zztiger123) [inline docs_espressif/zh_CN/commands.rst:28]:
```suggestion
     - 从 ESP-IDF 资源管理器 ``文档搜索结果`` 选项卡中清除所有搜索结果。
```

**Review by zztiger123 — APPROVED** [review]

**Pull request merged**